components/GitHubAccelerator.js

```diff
@@ -35,7 +35,7 @@ export default function GitHubAccelerator() {
             <div className="mt-6 flex flex-col justify-stretch space-y-3 sm:flex-row sm:space-x-4 sm:space-y-0">
               <Link
                 href="http://accelerator.github.com"
-                className="inline-flex justify-center rounded-md bg-white px-3 py-2 text-sm font-semibold text-primary-high dark:text-primary-high shadow-sm ring-1 ring-inset ring-primary-low-medium hover:bg-primary-low dark:hover:bg-slate-200 cursor-pointer"
+                className="inline-flex justify-center rounded-md bg-primary-low px-3 py-2 text-sm font-semibold text-primary-high dark:text-primary-high shadow-sm ring-1 ring-inset ring-primary-low/80 hover:bg-primary-low/95 dark:hover:bg-primary-low/95 cursor-pointer"
```

Comment:
Could you please tell what does this mean and how you come up with this? I want to know more about `/80` and `/95` thing.

`ring-primary-low-medium` to `ring-primary-low/80`
`bg-slate-200` to `bg-primary-low/95`

Reply:
Sure, I studied the previous code and understood the the logic behind the styling first, then implemented the change as per the tailwind.config.js requirements.

i.e 
**before**: `bg-white/5` , changing the numerical value in turn changes the shade of the color preceding it.

**after**: `bg-primary-low/85` (values starting ~ 5 to 100 will give different shades of the color associated with -primary-low (or any of the configured colors) in the tailwind.config.js file)

Hope that helps.

Reply:
It changes the alpha of the colour which is the opacity not the actual shade. so `green-50/90` is `green-50` with 90 percent opacity

Reply:
you can learn more here https://tailwindcss.com/docs/text-color . under the changing opacity section

Reply:
I didn't realize there was a difference between shade and opacity.